modules/tinymce/src/core/main/ts/keyboard/HomeEndKeys.ts

```diff
@@ -8,14 +9,23 @@ import * as MatchKeys from './MatchKeys';
 import * as MediaNavigation from './MediaNavigation';
 
 const executeKeydownOverride = (editor: Editor, caret: Cell<Text>, evt: KeyboardEvent) => {
-  MatchKeys.execute([
+  const os = PlatformDetection.detect().os;
```

Comment:
This should be using the `Env` API instead of doing this. Basically for TinyMCE we should always use `Env` except for in tests.